PackageNamespacesExample/README.md

```diff
@@ -0,0 +1,8 @@
+# Package Namespaces example
+
+This is an example demonstrating the [package namespaces feature](https://github.com/NuGet/Home/blob/dev/proposed/2021/PackageNamespaces.md).
+At the time of this writing, only a limited restore functionality is supported.
+

```

Comment:
How about adding a link somewhere to https://github.com/NuGet/NuGet.Client/blob/dev-kartheekp-ms-dogfood-namespaces/NuGet.Config and explain our dogfooding efforts for namespaces feature?

Reply:
Added.